Integrations/integration-cyberark_AIM.yml

```diff
@@ -0,0 +1,183 @@
+description: Query CyberArk Application Identity Manager for accounts and credentials
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Port
+  name: port
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: AppID as configured in AIM
+  name: appid
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Folder to search in safe
+  name: folder
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Safe to search in
+  name: safe
+  defaultvalue: isFetchCredentials
+  type: 0
+  required: true
+- display: ""
+  name: isFetchCredentials
+  defaultvalue: "true"
+  type: 8
+  required: false
+script:
+  script: |-
+    var port = params.port;
+    var base = params.server.replace(/[\/]+$/, '') + (port ? (':' + port) : '') + '/AIMWebService/api/';
+    var insecure = params.insecure;
+    var proxy = params.proxy;
+    var appid = params.appid;
+
+    var sendRequest = function(method, url, args, allowOtherStatuses) {
+        var res = http(
+                base + url + (method === 'GET' ? encodeToURLQuery(args) : ''),
+                {
+                    Method: method,
+                    Body: method !== 'GET' ? JSON.stringify(args) : ''
+                },
+                insecure,
+                proxy
+            );
+        if ((allowOtherStatuses && res.StatusCode !== 400) && (res.StatusCode < 200 || res.StatusCode >= 300)) {
+            if (res.StatusCode === 404) {
+                return 'no results found';
```

Comment:
You expect an object if success, but string in case 0f 404, might be better to return undefined and check it instead of a string